.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,6 +42,11 @@ Briefly summarize what changes were made to accomplish the motivation above.
 This may be unnecessary for some PRs. Catch-all for detailed explanations about the implementation decisions and implications of the change.
 
 
+**Testing**
+
+List the steps for reviewers to test the changes.
```

Comment:
This is interesting, I never really test things when I review. Could you maybe add an example to illustrate what someone would say here?
I could also interpret this section as what did the author do to test that the changes worked, what do you think?

Reply:
Yeah, my instinct would be to describe the steps that I personally took to test the PR before submitting it, so that reviewers can do the same test and confirm the results. I'll come up with an example.

Reply:
Ok cool yea that sounds good. At a previous job they basically had a template where you could check how you tested it. 

Something like:

- [ ]  Unit tests created
- [ ]  Manually tested ( this is like e2e test, where you run it as in production)
- [ ]  Existing tests adequate, no new tests required
- [ ]  Untested

Just some thoughts, dont have strong opinions for or against this field in general